generators/client/templates/angular/src/main/webapp/app/shared/social/_social.component.ts

```diff
@@ -21,7 +21,9 @@ export class <%=jhiPrefixCapitalized%>SocialComponent implements OnInit {
     ) {}
 
     ngOnInit() {
+        <%_ if (enableTranslation) { _%>
```

Comment:
Oh, so *that's* how the magic works, without cluttering up the files with if statements everywhere. Will change shortly.